### PR TITLE
feat: adding disabled column in repo_syncs & adding bool logic

### PR DESCRIPTION
**Pull request by ramiroJCB**
Here we add  a boolean  column in repo_syncs and we also  add logic that handles when to sync

### CODE REVIEW

File: internal/db/queries.sql [hunk 1]
```diff
@@ -81,7 +81,7 @@ UPDATE mergestat.repo_sync_queue SET last_keep_alive = now() WHERE id = $1;
 
 -- name: EnqueueAllCompletedSyncs :exec
 INSERT INTO mergestat.repo_sync_queue (repo_sync_id, status)
-SELECT id, 'QUEUED' FROM mergestat.repo_syncs WHERE id NOT IN (SELECT repo_sync_id FROM mergestat.repo_sync_queue WHERE status = 'RUNNING' OR status = 'QUEUED')
+SELECT id, 'QUEUED' FROM mergestat.repo_syncs WHERE enabled=true AND id NOT IN (SELECT repo_sync_id FROM mergestat.repo_sync_queue WHERE status = 'RUNNING' OR status = 'QUEUED')
```
Comment (patrickdevivo) [inline internal/db/queries.sql:84]:
```suggestion
SELECT id, 'QUEUED' FROM mergestat.repo_syncs WHERE enabled AND id NOT IN (SELECT repo_sync_id FROM mergestat.repo_sync_queue WHERE status = 'RUNNING' OR status = 'QUEUED')
```

just a nit 😃, feel free to ignore. `enabled` is already a boolean so we don't need to do the comparison here

Comment (ramiroJCB) [reply internal/db/queries.sql:84]: I like it !

Comment (ramiroJCB) [reply internal/db/queries.sql:84]: lets do it

Comment (patrickdevivo) [reply internal/db/queries.sql:84]: just remember to regenerate with `sqlc` as well 😃

Comment (ramiroJCB) [reply internal/db/queries.sql]: Done :)

**Pull request merged**